lib/Models/LegendHelper.js

```diff
@@ -466,18 +505,32 @@ function buildLegendProps(legendHelper, colorMap) {
         // Reverse the color bins so that the first one appears at the top, not the bottom.
         var reversedBinColors = binColors.slice(0, tableColumn.uniqueValues.length);
         reversedBinColors.reverse();
-        var items = reversedBinColors.map(function(bin, index) {
-            var label = tableColumn.uniqueValues[reversedBinColors.length - 1 - index];
-            // It would probably be cleaner to combine the binColor calculation and this calculation, so we don't have to do this test in each place.
-            if ((index === 0) && (tableColumn.uniqueValues.length > binColors.length)) {
-                label = defaultOtherEnumLabel;
-            }
-            return {
-                // Get the right label, noting the bin colors are reversed.
-                title: defined(label) ? label : nullLabel,
-                color: convertColorArrayToCssString(bin.colorArray)
-            };
-        });
+
+        var items;
+        if (legendHelper._cycleEnumValues) {
+            items = [{
+                title: defaultVariousValuesEnumLabel,
+                multipleColors: binColors.map(function(bin) {
+                    return convertColorArrayToCssString(bin.colorArray);
+                })
+            }];
+        } else {
+            items = reversedBinColors.map(function(bin, index) {
+                var label = tableColumn.uniqueValues[reversedBinColors.length - 1 - index];
+                // It would probably be cleaner to combine the binColor calculation and this calculation, so we don't have to do this test in each place.
+                if (tableColumn.uniqueValues.length > binColors.length && legendHelper._cycleEnumValues) {
```

Comment:
That's leftover code from when I was putting "Various Values" as the label for each of the colors, rather than sticking all  the colors in one box.